### ISSUE TITLE
Fix Deploy CI

### ISSUE THREAD

**Issue by 0x4007** [ubiquity/ubiquibot-telegram#13]
> > > Code looks good but @rndquu do you know why the CI is not reading the App ID?
> > 
> > 
> > The current PR is created from a forked repo hence it doesn't have access to secrets
> > Secrets can be read if workflow is executed on the `workflow_run` event ([example](https://github.com/ubiquity/pay.ubq.fi/blob/9c0d92ddbdf900570f417fe918ede7422bcd68df/.github/workflows/continuous-deploy.yml#L4)) not on [push](https://github.com/ubiquity/telegram-ubiquibot/blob/5ebe11792a6dfb4d7370e6b048c0845da4f248bc/.github/workflows/cloudflare-deploy.yml#L4) or [pull_request](https://github.com/ubiquity/telegram-ubiquibot/blob/5ebe11792a6dfb4d7370e6b048c0845da4f248bc/.github/workflows/cloudflare-deploy.yml#L7)
> > TLDR;
> > This [code](https://github.com/ubiquity/telegram-ubiquibot/blob/5ebe11792a6dfb4d7370e6b048c0845da4f248bc/.github/workflows/cloudflare-deploy.yml#L3) should be refactored to:
> > ```
> > on:
> >   workflow_run:
> >     workflows: ["Conventional Commits"]
> >     types:
> >       - completed
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > This way when the "Conventional Commits" is passed successfully then the "Deploy to Cloudflare Worker" will run initiated by the `workflow_run` event which will have access to reading secrets.
> 
> This needs to be done across all of our repositories right?

No, this is already properly setup for https://github.com/ubiquity/ubiquity-dollar and https://github.com/ubiquity/pay.ubq.fi

But the [deploy workflow](https://github.com/ubiquity/telegram-ubiquibot/blob/development/.github/workflows/cloudflare-deploy.yml) for the current repo should be refactored

_Originally posted by @rndquu in https://github.com/ubiquity/telegram-ubiquibot/issues/11#issuecomment-1680655298_
